versions.lock

```diff
@@ -156,6 +173,7 @@ org.apache.parquet:parquet-common:1.11.0 (2 constraints: 4c1e7785)
 org.apache.parquet:parquet-encoding:1.11.0 (1 constraints: ca0efb64)
 org.apache.parquet:parquet-format-structures:1.11.0 (3 constraints: 6e2bc836)
 org.apache.parquet:parquet-hadoop:1.11.0 (2 constraints: de1ac7b3)
+org.apache.parquet:parquet-hadoop-bundle:1.8.1 (1 constraints: 7a0bc7e4)
```

Comment:
This is the wrong version of Parquet. Can we remove this since we only need the metastore components?